Parser/Token/Token.php

```diff
@@ -26,6 +26,13 @@
 use Kadet\Highlighter\Parser\Validator\Validator;
 use Kadet\Highlighter\Utils\Helper;
 
+/**
+ * Class Token
+ *
+ * @package Kadet\Highlighter\Parser\Token
+ *
+ * @mixin Rule
```

Comment:
Makes phpstan understand that `__get()` (same for set and call) calls are proxied to the `Rule` class which in turn makes it realize where the fuck a variable in https://github.com/kadet1090/KeyLighter/blob/18d39a071b347328f326dbc95bba2a2ae6cac750/Parser/Token/LanguageToken.php#L46 is coming from